Project.toml

```diff
@@ -21,11 +21,11 @@ Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
 CSV = "0.5"
-FillArrays = "0.6"
-Optim = "0.18"
+FillArrays = "0.7"
+Optim = "0.19"
 Parameters = "0.10"
-RDatasets = "0.6.3"
-StatsBase = "0.31"
+RDatasets = "0.6"
```

Comment:
I chose `0.6.3` since basically I wanted to avoid the patched issues with CSV... I will defer to your judgement on whether we should use the minor or patch...

Reply:
I changed this because soon 0.6.4 will be released. I will then make it `"0.6.3, 0.6.4"` to follow the strict approach.